utilities/chemiscope.py

```diff
@@ -91,27 +91,23 @@ def chemiscope_input(meta, frames, projection, prediction,
 
     .. _`ase.Atoms`: https://wiki.fysik.dtu.dk/ase/ase/atoms.html
     '''
-
     data = {
         'meta': {
             'name': meta['name'],
-            'description': meta.get('description', None),
-            'authors': meta.get('authors', None),
-            'references': meta.get('references', None),
+            'description': meta.get('description', " "),
```

Comment:
why change from None to an empty string / empty lists? chemiscope should be able to deal with this

Reply:
I tryed to start chemiscope with file, where "description".."references" were None, and it returned a mistake. So I changed it in this way, no it work.

Reply:
ok, I would consider this a bug with chemiscope =) Could you share such a file with me (on slack or here) so that I can take a look?

Reply:
Yes, what file do you need?

Reply:
Yes, what file fo you need - notebook or a table?

Reply:
The generated json file that gives you an error when loaded on chemiscope!